wiki/en/Directories.md

```diff
@@ -1,22 +1,22 @@
 ---
 layout: wiki
-title: "Custom Directories"
+title: "Directories"
 lang: "en"
-permalink: "/wiki/Custom-Directories"
+permalink: "/wiki/Directories"
 ---
 
 {% include breadcrumb.html root="More" branch1="Server Administration" branch1-url="Running-a-Server" %}
 
 
-# Running a Custom Directory
+# Running a Directory
 
 This is a specialised Jamulus Server configuration, as described in [Server Types](Running-a-Server#server-types).
 
-To view Servers listed by a Custom Directory, users must enter the address of that Directory in their Client's Settings > Advanced Setup > Custom Directories field. Multiple addresses can be added in this way if needed. Custom Directories will then appear in the Directory drop-down list on their Connect window. Custom Directories otherwise work for Clients in the same way as Public Directories, displaying a list of Servers registered with them.
+To view Servers listed by a Directory, users must enter the address of that Directory in their Client's Settings > Advanced Setup > Directories field. Multiple addresses can be added in this way if needed. Directories will then appear in the Directory drop-down list on their Connect window. Directories otherwise work for Clients in the same way as the built-in Directories, displaying a list of Servers registered with them.
```

Comment:
Throughout this paragraph, "Custom" is correct because it differentiates from the "Public" Directories that are built into the client.  "Custom" _means_ "one you have to add by hand" for Directories.

So: Directory is _either_ "Public" ("built in") or "Custom" (entered by the user).  That's what we mean by "Public" and "Custom".

Reply:
Ah OK, will revert that then.